assessments/projects/chaos-mesh/self-assessment.md

```diff
@@ -0,0 +1,296 @@
+# Chaos Mesh Self-assessment
+
+Authors: Yash Mathur (@mathur-14), Aditya Deepak Joshi (@AdityaDJoshi), Ushan Abeysekera (@ua2047), Jia Huang (@iamgeeyuh)
+
+Contributors/Reviewers: Pranava Kumar Vemula (@Rana-KV), Eddie Knight (@eddie-knight)
+
+The Self-assessment is the initial document for Chaos Mesh to begin thinking about the security of the project, determining gaps in its security, and preparing any security documentation for their users.
+
+## Table of contents
+
+- [Self-assessment](#self-assessment)
+- [Self-assessment outline](#self-assessment-outline)
+  - [Table of contents](#table-of-contents)
+  - [Metadata](#metadata)
+    - [Security links](#security-links)
+  - [Overview](#overview)
+    - [Background](#background)
+    - [Actors](#actors)
+    - [Actions](#actions)
+    - [Goals](#goals)
+    - [Non-goals](#non-goals)
+  - [Self-assessment use](#self-assessment-use)
+  - [Security functions and features](#security-functions-and-features)
+  - [Project compliance](#project-compliance)
+  - [Secure development practices](#secure-development-practices)
+  - [Security issue resolution](#security-issue-resolution)
+  - [Appendix](#appendix)
+
+## Metadata
+
+|                   |                                                                                                                                   |
+| ----------------- | --------------------------------------------------------------------------------------------------------------------------------- |
+| Software          | https://github.com/chaos-mesh/chaos-mesh                                                                                          |
+| Security Provider | No, Chaos Mesh offers various types of fault simulation and has an enormous capability to orchestrate fault scenarios.            |
+| Languages         | Go, Typescript, Javascript, Shell                                                                                                 |
+| SBOM              | Chaos Mesh automatically generates SBOMs as assets with each release: https://github.com/chaos-mesh/chaos-mesh/releases/latest    |
+|                   |                                                                                                                                   |
+
+### Security links
```

Comment:
This space is also meant to include their vulnerability disclosure, incident response processes, default configurations, and anything else that can supplement security considerations. Please revisit and include them. In the event they aren't available, kindly mark them as not available

Reply:
Understood, we added links to the Vulnerability disclosure policy, default configurations, versions supported (for security updates) as well as the process that one has to go through for reporting a vulnerability.